### DISCUSSION TITLE
Access Denied

### DISCUSSION THREAD

**Discussion by zuihou** [minio/minio#16614] [Q&A] [answered]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Expected Behavior
Normal access

## Current Behavior
Access console prompt "Access Denied"

## Steps to Reproduce (for bugs)
<img width="1691" alt="image" src="https://user-images.githubusercontent.com/8500410/218623904-008f66df-4847-44d5-ac6b-ce0b790667b9.png">

## Your Environment

* Version used (`minio --version`):
```
minio version RELEASE.2023-02-10T18-48-39Z (commit-id=d0f4cc89a5d0ca45e7ba00fbb7733c2fead3f44e)
Runtime: go1.19.4 linux/amd64
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Copyright: 2015-2023 MinIO, Inc.
```
* Server setup and configuration:
```
nohup /home/pjjs/minio/minio server --config-dir /home/pjjs/minio/config /home/pjjs/minio_data --address 172.26.3.90:9000 --console-address 172.26.3.90:9090  > /home/pjjs/minio/minio.log 2>&1 &
```
* Operating System and version (`uname -a`):
```
Linux localhost.localdomain 5.10.0-4.uelc20.x86_64 #1 SMP Tue Feb 8 15:27:25 CST 2022 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by harshavardhana** [comment, marked as answer]
@zuihou looks like your login user doesn't have bucket list permissions.

**Comment by zuihou** [comment]
1.  start-up
```
export MINIO_ROOT_USER=admin
export MINIO_ROOT_PASSWORD="admin123"

nohup /home/pjjs/minio/minio server --config-dir /home/pjjs/minio/config /home/pjjs/minio_data2 --address 172.26.3.90:9000 --console-address 172.26.3.90:9090  > /home/pjjs/minio/minio.log 2>&1 &
```

2. Log in using "admin" 
3. Create a new bucket "aaaaa" 
4. visit "http://172.26.3.230:9090/browser/aaaaa" 
5. Access console prompt "Access Denied"

**Comment by harshavardhana** [comment]
@zuihou you need to check your MinIO logs you may have some disk permissions issue here. 

```
#!/bin/bash

export MINIO_ROOT_USER="admin"
export MINIO_ROOT_PASSWORD="admin123"

minio server /tmp/freshdata
```

```
~/run-minio.sh 
Formatting 1st pool, 1 set(s), 1 drives per set.
WARNING: Host local has more than 0 drives of set. A host failure will result in data becoming unavailable.
MinIO Object Storage Server
Copyright: 2015-2023 MinIO, Inc.
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Version: DEVELOPMENT.2023-02-14T08-21-17Z (go1.19.4 linux/amd64)

Status:         1 Online, 0 Offline. 
API: http://192.168.1.12:9000  http://172.17.0.1:9000  http://172.18.0.1:9000  http://127.0.0.1:9000           
RootUser: admin 
RootPass: admin123 
Console: http://192.168.1.12:46519 http://172.17.0.1:46519 http://172.18.0.1:46519 http://127.0.0.1:46519      
RootUser: admin 
RootPass: admin123 

Command-line: https://min.io/docs/minio/linux/reference/minio-mc.html#quickstart
   $ mc alias set myminio http://192.168.1.12:9000 admin admin123

Documentation: https://min.io/docs/minio/linux/index.html
```

![image](https://user-images.githubusercontent.com/622699/218689146-ab9a555e-a3e4-44d4-ae04-7c7fbb6120d4.png)

No error here.

**Reply by zuihou** [reply]
After changing the browser, the problem was solved

**Reply by harshavardhana** [reply]
what browser were you using? @zuihou

**Reply by zuihou** [reply]
> what browser were you using? @zuihou
Microsoft Edge

**Comment by zuihou** [comment]
Another problem:

Can [minio-java](https://github.com/minio/minio-java) modify the metadata of existing objects ?

When uploading an object, metadata is not set. Can you modify it after uploading successfully?
